File: projekt/server/DataBaseAPI/DataBaseAPI/Models/Aufgaben.cs
using System;
using System.Collections.Generic;

namespace DataBaseAPI.Models
{

  //beinhaltet alle Variablen aus der Tabelle Aufgaben (Datenbank)
    public partial class Aufgaben
    {
        public Aufgaben()
        {
            NutzerAufgaben = new HashSet<NutzerAufgaben>();
        }

        public int AufgabenId { get; set; }
        public string Titel { get; set; }
        public string Beschreibung { get; set; }
        public int? ErstellerId { get; set; }
        public int? ProjektId { get; set; }
        public bool? Erledigt { get; set; }
        public DateTime? ErstellDatum { get; set; }
        public int? VorgaengerId { get; set; }
        public int? NachfolgerId { get; set; }
        public double? SollZeit { get; set; }
        public double? IstZeit { get; set; }
        public DateTime? Deadline { get; set; }

        public virtual ICollection<NutzerAufgaben> NutzerAufgaben { get; set; }
    }
}

File: projekt/src/app/shared/aufgaben.ts
import { TeilAufgabe } from './teil-aufgabe';
import { Kommentar } from './kommentar';

export class Aufgaben {
    aufgabenId:number;
    titel?: string;
    beschreibung?: string;
    erstellerId?: number;
    projektId?: number;
    sollZeit?: number;
    istZeit?: number;
    erledigt?: boolean;
    erstellDdatum?: string;
    vorgaengerId?: number;
    nachfolgerId?: number;
    hasChanged?: boolean;
    teilaufgaben?: TeilAufgabe[];
    kommentar?: Kommentar[];
    deadline?: string;
    //deadlineD?: Date;
    //ueberzeit?: boolean;
}
